karapace/protobuf/proto_normalizations.py

```diff
@@ -0,0 +1,220 @@
+        location=enum_constant.location,
+        name=enum_constant.name,
+        tag=enum_constant.tag,
+        documentation=enum_constant.documentation,
+        options=sorted_options,
+    )
+
+
+def enum_element_with_sorted_options(enum_element: EnumElement) -> EnumElement:
+    sorted_options = None if enum_element.options is None else list(sorted(enum_element.options, key=sort_by_name))
+    constants_with_sorted_options = (
+        None
+        if enum_element.constants is None
+        else [enum_constant_element_with_sorted_options(constant) for constant in enum_element.constants]
+    )
+    return EnumElement(
+        location=enum_element.location,
+        name=enum_element.name,
+        documentation=enum_element.documentation,
+        options=sorted_options,
+        constants=constants_with_sorted_options,
+    )
+
+
+def groups_with_sorted_options(group: GroupElement) -> GroupElement:
+    sorted_fields = (
+        None if group.fields is None else [type_field_element_with_sorted_options(field) for field in group.fields]
+    )
+    return GroupElement(
+        label=group.label,
+        location=group.location,
+        name=group.name,
+        tag=group.tag,
+        documentation=group.documentation,
+        fields=sorted_fields,
+    )
+
+
+def one_ofs_with_sorted_options(one_ofs: OneOfElement) -> OneOfElement:
+    sorted_options = None if one_ofs.options is None else list(sorted(one_ofs.options, key=sort_by_name))
+    sorted_fields = [type_field_element_with_sorted_options(field) for field in one_ofs.fields]
+    sorted_groups = [groups_with_sorted_options(group) for group in one_ofs.groups]
+
+    return OneOfElement(
+        name=one_ofs.name,
+        documentation=one_ofs.documentation,
+        fields=sorted_fields,
+        groups=sorted_groups,
+        options=sorted_options,
+    )
+
+
+def message_element_with_sorted_options(message_element: MessageElement) -> MessageElement:
+    sorted_options = None if message_element.options is None else list(sorted(message_element.options, key=sort_by_name))
+    sorted_neasted_types = [type_element_with_sorted_options(nested_type) for nested_type in message_element.nested_types]
+    sorted_fields = [type_field_element_with_sorted_options(field) for field in message_element.fields]
+    sorted_one_ofs = [one_ofs_with_sorted_options(one_of) for one_of in message_element.one_ofs]
+
+    return MessageElement(
+        location=message_element.location,
+        name=message_element.name,
+        documentation=message_element.documentation,
+        nested_types=sorted_neasted_types,
+        options=sorted_options,
+        reserveds=message_element.reserveds,
+        fields=sorted_fields,
+        one_ofs=sorted_one_ofs,
+        extensions=message_element.extensions,
+        groups=message_element.groups,
+    )
+
+
+def type_element_with_sorted_options(type_element: TypeElement) -> TypeElement:
+    sorted_neasted_types: List[TypeElement] = []
```

Comment:
```suggestion
    sorted_nested_types: List[TypeElement] = []
```

Reply:
Also in test files multiple times same typo

Reply:
I've learned that nested doesn't have the `a` :D

Reply:
I've learned that nested doesn't have the `a` :D